uniffi_bindgen/src/interface/types/mod.rs

```diff
@@ -62,6 +62,9 @@ pub enum Type {
     Optional(Box<Type>),
     Sequence(Box<Type>),
     Map(/* String, */ Box<Type>),
+    // Types generated by uniffi but defined in other crates.
+    // Associated: (type_name, crate_name)
+    Imported(String, String),
```

Comment:
I wonder if there's value in either making these named fields, or having a helper strict `ImportedType` with named fields and having this be `Imported(ImportedType)`.